src/app/core/config/config-fix.ts

```diff
@@ -23,10 +23,13 @@ export const defaultJsonConfig = {
         link: "/"
       },
       {
-        label: $localize`:Menu item:Children`,
-        icon: "child",
-        link: "/child"
+        entityType: "Child",
       },
+      // {
```

Comment:
let's change all menu entries in the config-fix.ts to the new entityType mode (if applicable; the admin and other special ones will stay as is)